spec/forms/register_nonprofit_form/stripe_account_form_spec.rb

```diff
@@ -1,61 +1,54 @@
 # License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
 require "rails_helper"
 
-RSpec.describe RegisterNonprofitForm::StripeAccountForm, :type => :model do
-
-  around(:each) do |ex|
+RSpec.describe RegisterNonprofitForm::StripeAccountForm, type: :model do
+  around do |ex|
     StripeMockHelper.mock do
       ex.run
     end
   end
 
-  context 'validation' do
-    it 'accepts a nonprofit' do
+  describe "validation" do
+    it "accepts a nonprofit" do
       form = described_class.new(nonprofit: build(:nonprofit))
       expect(form).to be_valid
     end
 
-    it 'rejects a string' do
+    it "rejects nil" do
       form = described_class.new(nonprofit: nil)
-      expect(form).to_not be_valid
+      expect(form).not_to be_valid
```

Comment:
I'm not really a huge fan of switching from `to_not` to `not_to`. It's not a huge issue though.

Reply:
I think the linter recommends `not_to` so it is obvious when you scan it you don't miss the 'not' part after the to...